From 5d5eb5e14fa282f47405070fe22aa6a10f753e92 Mon Sep 17 00:00:00 2001
From: Iain Sandoe <iain@sandoe.co.uk>
Date: Sat, 17 Oct 2020 16:53:36 +0100
Subject: [PATCH] testsuite, Darwin, Arm64 : Skip two unsupported tests.

The compare function bodies code is ELF-specific at present
skip the tests until we can make a Mach-O version.

gcc/testsuite/ChangeLog:

	* g++.target/aarch64/no_unique_address_1.C: Skip.
	* g++.target/aarch64/no_unique_address_2.C: Skip.

(cherry picked from commit e3ac96c94b404c47974946d09e474d995c6757a1)
Signed-off-by: Kirill A. Korinsky <kirill@korins.ky>
---
 gcc/testsuite/g++.target/aarch64/no_unique_address_1.C | 2 +-
 gcc/testsuite/g++.target/aarch64/no_unique_address_2.C | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git gcc/testsuite/g++.target/aarch64/no_unique_address_1.C gcc/testsuite/g++.target/aarch64/no_unique_address_1.C
index 5fc68ea5d6d..5faf915fa54 100644
--- gcc/testsuite/g++.target/aarch64/no_unique_address_1.C
+++ gcc/testsuite/g++.target/aarch64/no_unique_address_1.C
@@ -1,5 +1,5 @@
 /* { dg-options "-std=c++11 -O -foptimize-sibling-calls -fpeephole2" } */
-/* { dg-final { check-function-bodies "**" "" "" { target lp64 } } } */
+/* { dg-final { check-function-bodies "**" "" "" { target { lp64 && { ! aarch64*-*-darwin* } } } } } */
 
 struct X { };
 struct Y { int : 0; };
diff --git gcc/testsuite/g++.target/aarch64/no_unique_address_2.C gcc/testsuite/g++.target/aarch64/no_unique_address_2.C
index f0717133ccd..322ec127c79 100644
--- gcc/testsuite/g++.target/aarch64/no_unique_address_2.C
+++ gcc/testsuite/g++.target/aarch64/no_unique_address_2.C
@@ -1,5 +1,5 @@
 /* { dg-options "-std=c++17 -O -foptimize-sibling-calls -fpeephole2" } */
-/* { dg-final { check-function-bodies "**" "" "" { target lp64 } } } */
+/* { dg-final { check-function-bodies "**" "" "" { target { lp64 && { ! aarch64*-*-darwin* } } } } } */
 
 struct X { };
 struct Y { int : 0; };
-- 
2.42.1

